### PR TITLE
#11:create and style for thumb card and txt

### PR DESCRIPTION
**Pull request by Lanng21**
image

### CODE REVIEW

File: src/components/card-provement.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="card {{ classCard }}">
+  <figure>
+    <img
+      src="../assets/images/photo.png"
+      alt="pl"
+      width="320px"
```
Comment (baonguyen26) [inline src/components/card-provement.html]: why set width and height for image?

File: src/components/card-provement.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div class="card {{ classCard }}">
+  <figure>
+    <img
+      src="../assets/images/photo.png"
```
Comment (baonguyen26) [inline src/components/card-provement.html]: all components have an image like this?

File: src/stylesheets/components/_proment-sec.scss [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        height: 144px;
+      }
+
+      p {
+        font-family: $Second-font;
+        line-height: 26px;
+        font-weight: 500;
+        font-size: 19px;
+        font-style: normal;
+        width: 322px;
```
Comment (baonguyen26) [inline src/stylesheets/components/_proment-sec.scss]: do not set width, height like this

File: src/stylesheets/components/_slider.scss [hunk 4]
```diff
@@ -2,14 +2,28 @@
...
+
+    @include mobile {
+      padding: 0 15px;
+      max-width: inherit;
+    }
 
     .main-girl {
       position: absolute;
-      right: 0;
+      right: 30px;
```
Comment (baonguyen26) [inline src/stylesheets/components/_slider.scss]: Double check this style, I think it not suitable

File: src/stylesheets/pages/home/_reach-customer-sec.scss [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  padding: 150px 0;
+  @include tablet {
+    padding: 60px 0;
+  }
+  .container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    max-width: 1140px;
```
Comment (baonguyen26) [inline src/stylesheets/pages/home/_reach-customer-sec.scss]: why does this place set the style of the container?

**Review by haunguyen15-tt — APPROVED** [review]

**Pull request merged**